libraries/AP_AdvancedFailsafe/AP_AdvancedFailsafe.cpp

```diff
@@ -275,6 +296,11 @@ AP_AdvancedFailsafe::check(uint32_t last_heartbeat_ms, bool geofence_breached, u
                 mission.set_current_cmd(_saved_wp);            
                 _saved_wp = 0;
             }
+        } else if (terminate_delay_gps) {
+            if (!_terminate) {
+               gcs().send_text(MAV_SEVERITY_CRITICAL, "TERM_DLY GPS Long Failsafe: Terminating!");
+                _terminate.set_and_notify(1);
```

Comment:
I'm not a big fan of enabling this like this. It's a critical and if some on accident then it gets very expensive and dangerous. Can we cause a terminate only if the terminate is already enabled?